packages/big-design/src/components/Select/types.ts

```diff
@@ -6,6 +6,7 @@ import { ListItemProps } from '../List/Item';
 
 interface BaseSelect extends Omit<React.HTMLAttributes<HTMLInputElement>, 'children'> {
   action?: SelectAction;
+  description?: string;
```

Comment:
This can actually accept a `string | FormControlDescription`. See our `Input` implements it.